en/payloads/README.md

```diff
@@ -0,0 +1,109 @@
+# Payloads and Cameras
+
+PX4 supports a wide range of payloads and cameras.
+
+## Mapping Drones
+
+* [Camera triggering](../peripherals/camera.md) via GPIO out
+* [Camera triggering](../peripherals/camera.md) via PWM out
+* [Camera triggering](../peripherals/camera.md) via MAVLink out
+* [Camera timing](../peripherals/camera.md#camera_capture) feedback via hotshoe input
+
+## Cargo drones and alike: Servos / Outputs
+
+* Servo or GPIO triggering (via RC or via commands)
+
+### Example Mission (in QGC)
+
+Use MAV_CMD_DO_SET_ACTUATOR to trigger one of the payload actuators.
+
+### Example script (MAVSDK)
+
+This script sends a command to set the actuator and trigger the payload release on a servo:
+
+```
+#include <mavsdk/mavsdk.h>
+#include <mavsdk/plugins/mavlink_passthrough/mavlink_passthrough.h>
+#include <mavsdk/plugins/info/info.h>
+#include <chrono>
+#include <cstdint>
+#include <iostream>
+#include <future>
+#include <memory>
+
+using namespace mavsdk;
+
+void send_actuator(MavlinkPassthrough& mavlink_passthrough,
+        float value1, float value2, float value3);
+
+int main(int argc, char **argv)
+{
+    Mavsdk mavsdk;
+    std::string connection_url;
+    ConnectionResult connection_result;
+    float value1, value2, value3;
+
+    if (argc == 5) {
+        connection_url = argv[1];
+        connection_result = mavsdk.add_any_connection(connection_url);
+        value1 = std::stof(argv[2]);
+        value2 = std::stof(argv[3]);
+        value3 = std::stof(argv[4]);
+    } 
+
+    if (connection_result != ConnectionResult::Success) {
+        std::cout << "Connection failed: " << connection_result << std::endl;
+        return 1;
+    }
+
+	bool discovered_system = false;
+    mavsdk.subscribe_on_new_system([&mavsdk, &discovered_system]() {
+		const auto system = mavsdk.systems().at(0);
+
+		if (system->is_connected()) {
+			std::cout << "Discovered system" << std::endl;
+			discovered_system = true;
+		}
+	});
+
+	std::this_thread::sleep_for(std::chrono::seconds(2));
+
+	if (!discovered_system) {
+        std::cout << "No device found, exiting." << std::endl;
+        return 1;
+    }
+
+	std::shared_ptr<System> system = mavsdk.systems().at(0);
```

Comment:
@julianoes Can you confirm my understanding of this block of code?
The first line just grabs the first system we connect to. The following lines then check all the connected systems for a specific hardware UID, and if one is found that is used instead. 

So my assumption is that this code allows us to make sure by UID we are connecting to the specific vehicle we want to, if it is connected. Is that the intent? 

If we care that much, shouldn't we pass that in as a value to the code and exit if we don't find it?

Further, do you have similar code in the MAVSDK docs? I see in https://mavsdk.mavlink.io/develop/en/guide/system_information.html#query-system-information we refer to this `.second` attribute but it isn't clear what it is/where it comes from (perhaps it is C++ that is new to me).

Reply:
I will create a proper example once the PR is in, and then I can help document that.